### DISCUSSION TITLE
Which events module does herdsman use ?

### DISCUSSION THREAD

**Discussion by PSTB** [Koenkk/zigbee2mqtt#16399] [General]
I notice there's an events.js in /node_modules on my system. 
Is this the one used by the herdsman controller, or is the built-in node.js module used ?

Thank you !

**Comment by PSTB** [comment]
No ideas, anyone ? :-)

**Reply by Koenkk** [reply]
You can check https://github.com/Koenkk/zigbee-herdsman/blob/master/package.json (it does not use `events.js` but the standard event emitter)

**Comment by PSTB** [comment]
Thank you Koen! I thought so, but wanted to get some confirmation. I'll now try to understand what that events.js in node_modules is being used for...
